src/components/Training/course/CourseWrapper.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import styled, { createGlobalStyle } from 'styled-components'
+import is from 'styled-is'
+
+import Layout from '../../../components/layout'
+
+const Wrapper = styled.div`
+  width: 100vw;
+  height: 100vh;
+  background-color: ${props => props.theme.colors.white};
+  position: fixed;
+  opacity: 0;
+  top: 0;
+  left: 0;
+  z-index: ${props => props.theme.zIndexes.modal};
+  transition: transform ${props => props.theme.animations.normal},
+    opacity ${props => props.theme.animations.normal};
+  transform: scale(0.7);
+  visibility: hidden;
+
+  ${is('visible')`
+    transform: scale(1);
+    opacity: 1;
+    visibility: visible;
+    overflow: scroll;
+  `};
+`
+
+const ModalStyles = createGlobalStyle`
+  body {
+    ${is('open')`
+      overflow: hidden;
+      position: absolute;
+    `}
+  }
+`
+
+const CourseWrapper = ({ location, content, children }) => (
+  <Layout location={location}>
+    <Wrapper visible={content}>
```

Comment:
do we need the `content` at all? Isn't this only rendered when navigated to?

Reply:
My guess is that if we navigated to this url, `visible` is always true, right? If so, why do we need it?

Reply:
Might be easiest to talk through on a call tomorrow?

Reply:
sure!